<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.2.335">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Ted Laderas, PhD â€“ retrospective</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1.6em;
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script>


<link rel="stylesheet" href="styles.css">
</head>

<body class="nav-fixed fullcontent">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg navbar-dark ">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container">
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">Ted Laderas, PhD</span>
    </a>
  </div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item">
    <a class="nav-link" href="./teaching.html">
 <span class="menu-text">Teaching</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./articles.html">
 <span class="menu-text">Articles</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./blog.html">
 <span class="menu-text">Blog</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./talks.html">
 <span class="menu-text">Talks</span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/laderast/"><i class="bi bi-github" role="img">
</i> 
 <span class="menu-text"></span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="https://twitter.com/tladeras/"><i class="bi bi-twitter" role="img">
</i> 
 <span class="menu-text"></span></a>
  </li>  
</ul>
              <div id="quarto-search" class="" title="Search"></div>
          </div> <!-- /navcollapse -->
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    
<!-- main -->
<main class="content" id="quarto-document-content">



<section id="retrospective-and-themes-for-2022" class="level1">
<h1>Retrospective and Themes for 2022</h1>
<p>Ted Laderas</p>
<p>2022 was a fairly busy year for me, not just with Academy engagements, but also in outreach and working on improving the user experience with effective workshops, courses, and developing optimal workflows for getting the main tasks on the platform done.</p>
<p>I've achieved this by observing user experiences and questions from the UKB RAP community, and from questions customers have asked during training sessions.</p>
<section id="improving-user-experience-through-establishing-best-practices" class="level2">
<h2 class="anchored" data-anchor-id="improving-user-experience-through-establishing-best-practices">Improving User Experience Through Establishing Best Practices</h2>
<p>One of my major passions in my work is to improve the overall user experience of the DNAnexus platform by establishing and suggesting effective workflows and best practices for getting tasks accomplished on the platform. These tasks include: executing batch jobs on the platform, working with RStudio Effectively on RAP, working with Docker, decoding pheno data from raw values, and using bash effectively on RAP.</p>
<p>Each of these accomplishments has stemmed from observing gaps in customer's knowledge in the platform. I believe that one of the biggest barriers to using the platform is still the feeling of impostor syndrome and I am passionate about giving customers the knowledge they need in formats that are helpful for them.</p>
<ul>
<li><p><strong>104 - Cloud Computing for HPC Users</strong>. This was a course and &gt; webinar that I developed with Anastazie Sedlakova and Scott &gt; Funkhouser. It attempts to directly map skills and concepts that &gt; incoming users already have with on-premise HPC and directly &gt; translates these skills to cloud HPC. It has been rated highly by &gt; both UKB RAP users and other customers.</p></li>
<li><p><strong>312 - JSON for the DNAnexus Platform</strong> - This is a course that I &gt; developed that fills in a lot of the gaps with utilizing JSON &gt; effectively on the DNAnexus platform. It helps users who are &gt; unfamiliar with JSON read and modify JSON for use in applets, but &gt; also in parsing JSON responses with the platform using jq, and &gt; doing batch submissions with jq.</p></li>
<li><p><a href="https://laderast.github.io/bash_for_bioinformatics/"><strong><u>Bash for &gt; Bioinformatics</u></strong></a> - &gt; In response to many of the challenges that I have seen with &gt; customers and app development and running jobs, I put together a &gt; "missing manual" that addresses the gaps in knowledge of bash &gt; scripting and where this knowledge is helpful on the platform. It &gt; has gotten positive reception not only on the UKB RAP community, &gt; but also by Solution Science as a resource they can point &gt; potential customers to.</p></li>
<li><p><strong>RStudio for RAP webinar</strong> - With Anastazie Sedlakova, I developed &gt; a webinar that outlined how to work reproducibly with RStudio on &gt; UKB RAP. This involved not only showing users how to use RStudio &gt; Projects, but also how to save software environments for &gt; reproducible research, and run RStudio/Bioconductor via Docker &gt; containers.</p></li>
<li><p><strong>Docker for RAP webinar</strong> - This was a webinar that I developed &gt; with Ondrej Klempir on how to effectively use Docker on UKB RAP &gt; for batch processing. It provides specific bash recipes for saving &gt; docker image files, extending docker images, and running batch &gt; jobs with Docker image files.</p></li>
<li><p><a href="https://laderast.github.io/dxhelper/"><u><strong>dxhelpe</strong>r</u></a> - &gt; this is an R Package I have developed to help R users with &gt; decoding Pheno data extracted from the new dx extract_dataset &gt; functionality in dx-toolkit. R users are a major percentage of UKB &gt; RAP users, and this package will helps them with their overall &gt; user experience with the pheno data.</p></li>
</ul>
</section>
<section id="improving-internal-training-and-communication" class="level2">
<h2 class="anchored" data-anchor-id="improving-internal-training-and-communication">Improving Internal Training and Communication</h2>
<p>I have helped onboard a number of our new employees through both Bootcamp and a formal set of courses. This was in direct response to Dick's call for more internal training specifically for our sales team.</p>
<ul>
<li><p><strong>Ongoing training with Alex Edwards</strong>. Alex and I continue to &gt; cross-train each other on various features of the platform, &gt; including Apollo specific features. He's now ready to train OFH &gt; alpha users on these features. I hope to learn more about the &gt; ingestion process next year.</p></li>
<li><p><strong>Trained 2 incoming groups on platform, including Apollo &gt; features.</strong> We have increased our overall training in our &gt; onboarding program, including incoming members of the Sales team.</p>
<ul>
<li><p>This has included frank discussions and addressing questions the &gt; Sales team has about the product, and helping them develop the &gt; language to effectively communicate about the platform.</p></li>
<li><p>I believe this has helped not only in increasing knowledge, but &gt; also communication between various groups at DNAnexus.</p></li>
</ul></li>
<li><p><strong>Gave Science Deep Dive talk on Teaching</strong>: <a href="https://docs.google.com/presentation/d/1RVuCnQy_sshmeB9Bc6s75PZJfpKOqVJ5CL_CD0KUR-E/edit?usp=sharing"><u>Using Cognitive &gt; Science to Make Training &gt; Stick</u></a></p></li>
<li><p><strong>Talk for xVantage Day:</strong> Using Quarto: &gt; <a href="https://laderast.github.io/qmd_rmd/#/title-slide"><u>https://laderast.github.io/qmd_rmd/</u></a></p></li>
<li><p><strong>Trained VN team on platform</strong>. As part of the Vietnam Team's &gt; onboarding, we delivered the Titan/Apollo courses to them.</p></li>
<li><p><strong>Outreach to other groups.</strong> I continue to work and communicate &gt; with members of Customer Care, Translational Medicine, Customer &gt; Success, UKB RAP, and Solution Science</p></li>
</ul>
</section>
<section id="outreach-to-new-audiences" class="level2">
<h2 class="anchored" data-anchor-id="outreach-to-new-audiences">Outreach to New Audiences</h2>
<p>I want others to effectively utilize the platform and accomplish work with it. To this end, I have helped with outreach to a number of new audiences with Ben Busby and the UKB RAP team. I</p>
<ul>
<li><p><strong>RAP getting started workshops.</strong> I sat in on 5 sessions that were &gt; a combination of workshop and Q&amp;A for new UKB RAP users. I was &gt; actively answering questions in the chat and providing links to &gt; material.</p></li>
<li><p><strong>HDR UK</strong> - I was part of a group of educators talking about TREs &gt; and their role in working with clinical data. Specifically I &gt; talked about the role of UKB RAP and how it fulfilled the &gt; requirements of a TRE (patient security) in spite of it being a &gt; cloud-based solution</p></li>
<li><p><strong>Imaging Workshop</strong> - I helped Renee Qian with setup and running &gt; the MATLAB container on UKB RAP for the imaging workshop, &gt; including showing her how to use dxFUSE for her demo.</p></li>
<li><p><strong>ISCB Academy</strong> - Ben and I presented various UKB RAP features as a &gt; workshop for ISCB Academy, and shared Bash for Bioinformatics as a &gt; resource for others.</p></li>
<li><p><strong>I developed and shared an R specific workflow</strong> for UKB RAP for &gt; exporting, decoding, and working with Pheno data using my dxhelper &gt; package. It was used for the in-person UKB RAP workshop this &gt; December.</p></li>
</ul>
</section>
<section id="section" class="level2">
<h2 class="anchored" data-anchor-id="section"></h2>
</section>
<section id="course-development-and-innovation" class="level2">
<h2 class="anchored" data-anchor-id="course-development-and-innovation">Course Development and Innovation</h2>
<p>As Academy, we continue to innovate and deliver effective training through our use of active learning exercises and other modalities.</p>
<p>One of the ongoing challenges is training customers on the different versions of the platforms (OFH Alpha/Beta, Optum Clinico-Genomics, UKB RAP, and the GA platform).</p>
<p>One of our goals is to integrate new features as shared by product as quickly into course materials as possible. Sometimes the material provided by product needs improvement in terms of user experience, and we aim to deliver improved notebooks so we can better teach the respective topic.</p>
<p>This is a list of the courses that I have actively developed this year:</p>
<table class="table">
<colgroup>
<col style="width: 27%">
<col style="width: 20%">
<col style="width: 50%">
</colgroup>
<thead>
<tr class="header">
<th><strong>Courses</strong></th>
<th><strong>Platform</strong></th>
<th><strong>Description of work</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><ul>
<li>**103 Cloud &gt; computing &gt; for</li>
</ul>
<blockquote class="blockquote">
<p>Scientists**</p>
</blockquote>
<ul>
<li><p><strong>103 Cloud &gt; computing &gt; for &gt; Scientists &gt; (On &gt; Demand)</strong></p></li>
<li><p><strong>103 Cloud &gt; computing &gt; for &gt; Scientists &gt; (RAP)</strong></p></li>
</ul></td>
<td>Core</td>
<td><ul>
<li><p>Development of webinar &gt; version (for RAP)</p></li>
<li><p>Development of on-demand &gt; course. Currently in &gt; testing.</p></li>
</ul></td>
</tr>
<tr class="even">
<td><ul>
<li><p><strong>104 &gt; DNAnexus &gt; for HPC &gt; Users</strong></p></li>
<li><p><strong>104 &gt; DNAnexus &gt; for HPC &gt; Users &gt; (RAP)</strong></p></li>
</ul></td>
<td>Core/RAP</td>
<td><ul>
<li><p>Development of material and &gt; webinar for RAP.</p></li>
<li><p>Course has also been adapted &gt; as a GA-specific course &gt; with active learning &gt; exercises showing both &gt; running single and batch &gt; jobs using the dx-toolkit.</p></li>
</ul></td>
</tr>
<tr class="odd">
<td><ul>
<li><strong>272 Complex &gt; Cohort &gt; Analysis</strong></li>
</ul></td>
<td>Core</td>
<td><ul>
<li><p>Exercises have been updated &gt; with complete answers &gt; (including cohort diagrams) &gt; and opportunities for &gt; discussion</p></li>
<li><p>Exercises have been added for &gt; cohort combine &gt; functionality of Cohort &gt; Browser</p></li>
</ul></td>
</tr>
<tr class="even">
<td><ul>
<li>340R</li>
</ul>
<blockquote class="blockquote">
<p>Introduction</p>
</blockquote>
<pre><code>&gt; to WDL for
&gt; DNAnexus
&gt; users (RAP)</code></pre></td>
<td>RAP</td>
<td><ul>
<li>Adapted existing WDL course &gt; for a 2 session webinar for &gt; RAP.</li>
</ul></td>
</tr>
<tr class="odd">
<td><ul>
<li>Docker for &gt; UKB RAP &gt; Webinar</li>
</ul></td>
<td>RAP</td>
<td><ul>
<li><p>Best practices for running &gt; docker images within &gt; applications, including</p>
<ul>
<li><p>Pulling/saving docker &gt; image files</p></li>
<li><p>Using docker image files &gt; in applets</p></li>
<li><p>Extending docker images &gt; using interactive mode &gt; and Dockerfiles</p></li>
</ul></li>
</ul></td>
</tr>
<tr class="even">
<td><ul>
<li>302 (On &gt; Demand) - &gt; Command &gt; Line for &gt; UKB RAP</li>
</ul></td>
<td>RAP</td>
<td><ul>
<li>Development of videos and &gt; exercises showing effective &gt; ways to work with the &gt; command-line on RAP</li>
</ul></td>
</tr>
<tr class="odd">
<td><ul>
<li>270 &gt; JupyterLabs &gt; update</li>
</ul></td>
<td>Core</td>
<td><ul>
<li><p>The JupyterLabs course has &gt; been updated to pheno data &gt; extraction using dx &gt; extract_data. R and Python &gt; Specific notebooks have &gt; been added for working with &gt; the raw extracted data.</p></li>
<li><p>Notebooks have also been &gt; added to show how to &gt; extract the data using a &gt; Spark dxjupyter cluster.</p></li>
</ul></td>
</tr>
<tr class="even">
<td><ul>
<li><p><strong>101 Optum</strong></p></li>
<li><p><strong>170 Optum</strong></p></li>
<li><p><strong>270 Optum</strong></p></li>
</ul></td>
<td>Optum Cli nicogenomics Platform</td>
<td><ul>
<li><p>The Apollo training program &gt; was adapted to incorporate &gt; Clinicogenomic specific &gt; features and restrictions &gt; (specifically download &gt; restrictions).</p></li>
<li><p>This especially impacts how &gt; users use JupyterLab &gt; (download restrictions &gt; prevent opening notebooks &gt; directly on the platform)</p></li>
</ul></td>
</tr>
<tr class="odd">
<td><ul>
<li><p><strong>101 OFH</strong></p></li>
<li><p><strong>170 OFH</strong></p></li>
<li><p><strong>270 OFH</strong></p></li>
</ul></td>
<td>OFH Alpha platform</td>
<td><ul>
<li>Apollo training program was &gt; adapted for OFH Alpha &gt; specific features, &gt; including &gt; restricted/unrestricted &gt; projects and &gt; download/upload &gt; restrictions</li>
</ul></td>
</tr>
<tr class="even">
<td><ul>
<li>**312</li>
</ul>
<blockquote class="blockquote">
<p>Introduction</p>
</blockquote>
<pre><code>&gt; to JSON**</code></pre></td>
<td>All</td>
<td><ul>
<li><p>Development of course &gt; exploring the use of JSON &gt; on the platform.</p></li>
<li><p>Exercises processing JSON &gt; responses using jq have &gt; been added</p></li>
<li><p>Course to be used as a &gt; prerequisite for App &gt; Building course (310) where &gt; necessary</p></li>
</ul></td>
</tr>
<tr class="odd">
<td><ul>
<li><p><strong>313 (RAP) - &gt; Running &gt; Apps on UKB &gt; RAP</strong></p></li>
<li><p><strong>314 (RAP) - &gt; Building &gt; Apps on UKB &gt; RAP</strong></p></li>
</ul></td>
<td>RAP</td>
<td><ul>
<li><p>Added exercises in running &gt; existing apps on RAP, &gt; including batch mode</p></li>
<li><p>Added exercises in building &gt; applets that map closely to &gt; the overall development &gt; process</p></li>
</ul></td>
</tr>
</tbody>
</table>


</section>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "î§‹";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>