---
title: "Ted Laderas, PhD"
output: pdf_document
layout: page
---

*Assistant Professor*, 
Division of Bioinformatics and Computational Biology, 
Department of Medical Informatics and Clinical Epidemiology

*Researcher*, OHSU Knight Cancer Institute  
Oregon Health & Science University  
3181 SW Sam Jackson Park Road CR145  
Portland, OR 97239  
(503) 481-8470  
<laderast@ohsu.edu>  

**EDUCATION**

B.A., Chemistry, Reed College, Portland, OR 1998.  
M.S. Biomedical Informatics, Oregon Health & Science University, 2004.  
PhD, Biomedical Informatics, Oregon Health & Science University, 2014.

**RESEARCH STATEMENT**

My research interests are complex diseases, precision medicine,
applications of systems science (including network analysis and
modeling), and applying data integration to difficult and high-impact
translational research questions. These questions include immune system
profiling in both infectious disease (tuberculosis) and Acute Myeloid Leukemia,
understanding drug sensitivity in the context of multiple cancer types
(AML, Colorectal, Breast and Head and Neck Cancer), and quantifying
expression differences in alcoholic preference. I have worked with a
large number of datatypes (high-throughput immunophenotyping,
proteomics, expression, genomic, and functional drug screen data) and
have focused on methods and frameworks integrating these datatypes
within the biological and clinical context of these translational
research questions. My training in biomedical informatics as a master’s
student in Biomedical Informatics, as an NLM Predoctoral Fellow, and as
a NLM Postdoctoral fellow has enabled me to communicate with a wide
variety of collaborators by giving me a strong background in Cancer
Biology, Software Development, and Clinical Systems. Additionally, I am
a strong advocate for Open Science initiatives, most notably the effort
for reproducibility in scientific analysis. To this end, I have
developed multiple novel software pipelines that transparently process
data from raw data to through the final stages of analysis.

**SELECTED RESEARCH PUBLICATIONS**

1. *Teaching data science fundamentals through realistic synthetic clinical cardiovascular data.* **Ted Laderas**, Nicole Vasilevsky, Bjorn Pederson, Shannon McWeeney, Melissa Haendel, and David Dorr. Submitted to JAMIA. BioRxiv Link: https://www.biorxiv.org/content/early/2017/12/12/232611. 2017. Contribution: First author: helped conceive study, designed bayesian network, developed course material based on dataset. 

2. *Integrated functional and mass spectrometry-based flow cytometric phenotyping to describe the immune microenvironment in acute myeloid leukemia*. Adam J Lamble, Matt Dietz, **Ted Laderas**, Shannon McWeeney, Evan F Lind. Journal of Immunological Methods. 2017. Contribution: developed informatics and visualization methodology.

3.  *Probabilistic Graphical Models for Systems Biology*. **Ted
    Laderas**, Aurora Blucher, Guanming Wu and Shannon McWeeney.
    Awaiting submission. Contribution: first author and conception
    of manuscript.

4.  [*A Network-Based Model of Oncogenic Collaboration to Predict Drug
    Sensitivity*.](http://journal.frontiersin.org/article/10.3389/fgene.2015.00341/abstract)
    **Ted Laderas**, Laura Heiser and Kemal Sonmez. Featured article in
    Frontiers in Genetics. 2015. Contribution: First author, developer
    of methodology and visualization framework.

5. [*Mass Cytometry As a Modality to Identify Candidates for Immune Checkpoint Inhibitor Therapy within Acute Myeloid Leukemia*](http://www.bloodjournal.org/content/128/22/2829?sso-checked=true). Adam Lamble, Yoko Kosaka, Fei Huang, A Kate Sasser, Homer Adams III, Cristina E. Tognon, **Ted Laderas**, Shannon K. McWeeney, Marc M. Loriaux, Jeffrey W. Tyner, Brian J. Druker and Evan F. Lind. Blood, 2016. Contribution: contributed to panel development of CyTOF.

6.  [*The Consensus Molecular Subtypes of Colorectal
    Cancer*.](https://www.dropbox.com/s/p92dn5buctlyqpc/nm.3967.pdf?dl=0)
    123 citations, Impact Factor 30.36. Justin Guinney, Rodrigo
    Dienstmann, Xin Wang, Aurélien de Reyniès, Andreas Schlicker,
    Charlotte Soneson, Laetitia Marisa, Paul Roepman, Gift Nyamundanda,
    Paolo Angelino, Brian M. Bot, Jeffrey S. Morris, Iris Simon, Sarah
    Gerster, Evelyn Fessler, Felipe de Sousa e Melo, Edoardo Missiaglia,
    Hena Ramay, David Barras, Krisztian Homicsko, Dipen Maru,
    Ganiraju C. Manyam, Bradley Broom, Valerie Boige, **Ted Laderas**,
    Ramon Salazar, Joe W. Gray, Douglas Hanahan, Josep Tabernero, Rene
    Bernards, Stephen H. Friend, Pierre Laurent-Puig, Jan P. Medema,
    Anguraj Sadanandam, Lodewyk Wessels, Mauro Delorenzi, Scott Kopetz,
    Louis Vermeulen, and Sabine Tejpar. Nature Medicine. 2015.
    Contribution: mapped and analyzed OMICs data to consensus cancer subtypes.

7.  [*Between Pathways and Networks lies
    Context*](https://www.dropbox.com/s/70z4yp3k36y9bmu/laderast_between_pathways.pdf?dl=0).
    Impact Factor: 1.76. **Ted Laderas**, Guanming Wu, and
    Shannon McWeeney. Science Progress. 2015. Contribution: first
    author, conceptualized and wrote manuscript.
    [doi:10.3184/003685015X14368898634462](http://dx.doi.org/10.3184/003685015X14368898634462)

8.  [*Computational detection of alternative exon
    usage*.](https://www.dropbox.com/s/t9ic1rpirja4xqv/computational-exon-laderas.pdf?dl=0)
    8 citations, Impact Factor: 3.398. **Ted Laderas**, Nicole A Walter,
    Michael Mooney, Kristina Vartanian, Priscila Darakjian, Kari Buck,
    Chris Harrington, John Belknap, Robert Hitzemann, and
    Shannon McWeeney. Frontiers in Neurogenomics. 2011. Article 69.
    PMID 21625610. Contribution: First author, developer of methodology.
    [doi:10.3389/fnins.2011.00069](http://dx.doi.org/10.3389/fnins.2011.00069)

9.  [*High throughput sequencing in mice: a platform comparison
    identifies a preponderance of cryptic
    SNPs*.](https://www.dropbox.com/s/2v0h7ufxu1664yt/preponderance-SNPs-walter-laderas.pdf?dl=0)
    19 Citations, Impact Factor 3.87. Nicole A Walter, Daniel Bottomly,
    **Ted Laderas**, Michael Mooney, Priscila Darakjian, Robert P
    Searles, Christina Harrington, Shannon K McWeeney, Robert Hitzemann,
    Kari J Buck. BMC Genomics. 2009 Aug 17;10:379. Contribution: helped
    develop methodology of manuscript, conceptualization.
    [doi:10.1186/1471-2164-10-379](http://dx.doi.org/10.1186/1471-2164-10-379)

10.  *TandTRAQ: An open-source tool for integrated protein identification
    and quantitation*. 14 citations, Impact Factor 5.766. **Ted
    Laderas**, Cory Bystrom, Debra McMillen, Guang Fan and
    Shannon McWeeney. Bioinformatics. 2007. Contribution: first author,
    designed software framework.
    [doi:10.1093/bioinformatics/btm467](http://dx.doi.org/10.1093/bioinformatics/btm467)

11.  [*A consensus framework for exploring microarray data using multiple
    clustering
    methods*.](https://www.dropbox.com/s/v56bomf1ijhmds3/Laderas_McWeeney07.pdf?dl=0)
    16 citations. **Ted Laderas** and Shannon McWeeney. OMICS: A Journal
    of Integrative Biology. OMICS: A Journal of Integrative Biology. 2007. 116-128. Contribution: first author, developed methodology for comparison analysis.
    [doi:10.1089/omi.2006.0008](http://dx.doi.org/10.1089/omi.2006.0008)

**RESEARCH SOFTWARE (Sole Developer unless specified)**

1. *cvdRiskData*. Synthetic dataset for teach fundamentals in data science. Available as an R package at: https://github.com/laderast/cvdRiskData. 

1. *RBootcamp (tidyverse version)*. DataCamp active exercises for learning introductory R using the `tidyverse`. With Jessica Minnier and Chester Ismay. Available for free at: https://www.datacamp.com/courses/rbootcamp <https:://doi.org/10.5281/zenodo.854727>

2.  *flowDashboard*. Visualization framework in R/Shiny and processing
    pipeline for CyTOF and flow cytometry data. Currently under embargo,
    waiting publication and submission. <https://doi.org/10.5281/zenodo.260421>

3.  *Surrogate Network Explorer for Head and Neck Cancer*. Interactive
    visualization framework for understanding network effects in Head
    and Neck cancer. Awaiting validation, publication, and submission.

4.  *R-Bootcamp: Introduction to R*. Script-based introductory MOOC
    to R. With contributions from Eric Leung, Dian Chase, Tracy Edinger,
    Clint Olson, and Gabrielle Chonoo.
    <http://dx.doi.org/10.5281/zenodo.13756>

5.  *Surrogate Mutation Explorer*. R/Shiny Interactive Application for
    exploring surrogate mutations.
    <http://dx.doi.org/10.5281/zenodo.13757>

6.  *SurrogateMutation*. R Package for mapping mutations and copy number
    alterations to networks and associated statistics.
    <http://dx.doi.org/10.5281/zenodo.17303>

7.  *Consense-Cluster*. R Package for comparing clustering output
    across algorithms. <http://dx.doi.org/10.5281/zenodo.17304>

8.  *ExonModelStrain*. RPackage for detecting alternative exon usage
    between two strains of mice.
    <http://dx.doi.org/10.5281/zenodo.17305>

9.  *TandTRAQ*. Perl Script for mapping iTRAQ protein quantitation to
    X!Tandem peptides.

**RESEARCH PRESENTATIONS/LECTURES (Invited Lectures are Indicated)**

1.  [*Data Science for Basic Scientists*](http://laderast.github.io/DS4BS/). Invited talk for OHSU Symposium on Educational Excellence. April 2017. [doi:10.6084/m9.figshare.4876811.v3](https://doi.org/10.6084/m9.figshare.4876811.v3)

2.  [*Moving From Big Data to Knowledge: Implications for the Health
    Care and Biomedical
    Sciences*](https://www.dropbox.com/s/ls6agg9uy2b6wp4/big-data-talk_laderas.pdf?dl=0)*.*
    Invited Lecture given for BioCatalyst Advanced Training workshop at
    Oregon Bioscience Association. June 2015.

3.  *Surrogate Mutations and Drug Sensitivity in Breast Cancer
    Cell Lines.* Plenary talk at NLM Training Conference, Pittburgh, PA,
    June 2014.

4.  *Connecting Genotypes to Drug Sensitivities in HER2 Positive Cancer
    Cell Lines.* Doctoral Defense, OHSU, March 2014.

5.  *Understanding Expression Differences in RPPA Data: Implications for
    ODE Models.* Presentation at Integrative Cancer Biology Program
    retreat 2012.

6.  *Connecting Genotypes to Drug Sensitivity in HER2+ Breast Cancer
    Cell Lines.* Lecture/Dissertation proposal given for PhD/Postdoc
    Meeting January 2012 and March 2012.

7.  *Connecting Genotypes to Drug Sensitivity in Breast Cancer
    Cell Lines.* Invited presentation at Integrative Cancer Biology
    Program retreat, 2011.

8.  [*What are Models For? Making Sense of Systems
    Biology*](https://www.dropbox.com/s/bi14mo1ahr4yod8/symposium-laderas.pptx?dl=0).
    Predoctoral Symposium. 2010.

9.  *Two kinds of Robustness.* Presentation given to Bioinformatics
    Development Group, 2010.

10.  *Portland Alcohol Research Center Scientific Advisory Board Review:
    Bioinformatics & Statistics.* Presentation given to PARC SAB,
    OHSU, 2009.

11. *An introduction to trend analysis and consensus modules for
    systems biology.* Presentation given with Sophia Jeng to
    Mathematical Modeling Core for Systems Virology, OHSU 2009.

12. *An introduction to Consensus Modules for Systems Biology.* Lecture
    given with Sophia Jeng to Bioinformatics Development Group,
    OHSU, 2008.

13. *An Introduction to Sloppy Systems Biology.* Lecture given to
    Bioinformatics Development Group, OHSU, 2008.

14. *Simulated and Synthetic Data for Benchmarking Algorithms.* Lecture
    given as part of Bioinformatics Methods class, OHSU, 2007.

15. *Introduction to QPACA Pathway Analysis Software and BioPAX.* Talk
    given at 2006 caBIG Annual Meeting, Washington DC.

16. *Consense: A software package for utilizing multiple clustering
    methods on Microarray Data.* Talk given at OHSU Student Research
    Forum, 2005.

**RESEARCH POSTERS/ABSTRACTS**

1.  *Getting your hands dirty with data*. **Ted Laderas**, Melissa
    Hanedel, Bjorn Pederson, Jackie Wirz, William Hersh, David A. Dorr,
    Shannon McWeeney. Big Data to Knowledge (BD2K) All Hands meeting,
    November 2016. [doi:10.6084/m9.figshare.4235594.v1](https://doi.org/10.6084/m9.figshare.4235594.v1)

2.  *Get Real: A synthetic dataset illustrating clinical and
    genetic covariates.* **Ted Laderas**, Nicole Vasilevsky, Melissa
    Haendel, Shannon McWeeney and David A. Dorr. BD2K All Hands meeting,
    November 2016. [doi:10.6084/m9.figshare.4239959.v1](https://doi.org/10.6084/m9.figshare.4239959.v1)

3.  *Mass Cytometry as a Modality to Identify Candidates for Immune
    Checkpoint Inhibitor Therapy within Acute Myeloid Leukemia.* Adam
    Lamble, Fei Huang, Kate Sasser, Homer Adams III, Cristina E. Tognon,
    **Ted Laderas**, Shannon McWeeney, Marc Loriaux, Jeffery W. Tyner,
    Brian J. Druker, and Evan F. Lind. American Society for Hematology,
    December 2016.

4.  *Enhanced VISTA Expression in a Subset of Patients with Acute
    Myeloid Leukemia*. Adam Lamble, Yoko Kosaka, Fei Huang, Kate Sasser,
    Homer Adams III, Cristina E. Tognon, **Ted Laderas**, Shannon
    McWeeney, Marc Loriaux, Jeffery W. Tyner, Brian J. Druker, and
    Evan F. Lind. American Society for Hematology, December 2016.

5.  *Improving Knowledge Discovery Through Development of Big Data to
    Knowledge Skills Courses and Open Educational Resources*. Nicole
    Vasilevsky, Shannon McWeeney, William Hersh, David A. Dorr, **Ted
    Laderas**, Jackie Wirz, Bjorn Pederson and Melissa Haendel. Poster for
    BICC Building 25th Anniversary.

6.  *Consensus molecular subtyping through a community of experts
    advances unsupervised gene expression-based disease classification
    and facilitates clinical translation.* Justin Guinney, Rodrigo
    Dienstmann, Xin Wang, Aurelien de Reynies, Andreas Schlicker,
    Charlotte Soneson, Laetitia Marisa, Paul Roepman, Gift Nyamundanda,
    Paolo Angelino, Brian Bot, Jeffrey S. Morris, Iris Simon, Sarah
    Gerster, Evelyn Fessler, Felipe de Sousa e Melo, Edoardo Missiaglia,
    Hena Ramay, David Barras, Krisztian Homicsko, Dipen Maru, Ganiraju
    Manyam, Bradley Broom, Valerie Boige, **Ted Laderas**, Ramon
    Salazar, Joe W. Gray, Josep Tabernero, Rene Bernards, Stephen
    Friend, Pierre Laurent-Puig, Jan P. Medema, Anguraj Sadanandam,
    Lodewyk Wessels, Mauro Delorenzi, Scott Kopetz, Louis Vermeulen and
    Sabine Tejpar. American Association for Cancer Research Conference,
    August 2015.

7.  *A Network-Based Model of Oncogenic Collaboration for Prediction of
    Drug Sensitivity.* **Ted Laderas**. RECOMB/ISCB Conference on
    Regulatory and Systems Genomics, November 2014.

8.  [*Integrating proteomics and genomic data to illuminate the effects
    of Lapatinib in HER2+
    Cells*](https://www.dropbox.com/s/y3rbkygjn41qsd6/poster-laderas-nlm-2013.pdf?dl=0).
    **Ted Laderas** and Kemal Sonmez. National Library of Medicine
    Training Conference, Utah, 2013. [doi:10.6084/m9.figshare.4829749.v1](https://doi.org/10.6084/m9.figshare.4829749.v1)

9.  [*Integrating to Interact: Using Reconstructability Analysis to
    Integrate Genomic and Proteomic Data to Predict Drug
    Sensitivity*](https://www.dropbox.com/s/nhixsztlg1oo4z6/poster-laderas-ismb-2012.pdf?dl=0).
    **Ted Laderas** and Kemal Sonmez. Intelligent Systems in Molecular
    Biology conference, 2012. [doi:10.6084/m9.figshare.4830007.v1](https://doi.org/10.6084/m9.figshare.4830007.v1)

10. *Tales from the Cryptic Snp: High Throughput Sequencing of DBA/2J
    and C57BL/6J mouse chromosome 1 reveals an abundance
    of polymorphisms.* Kari J. Buck, Nicole A.R. Walter, Daniel
    Bottomly, **Ted Laderas**, Michael Mooney, Priscila Darakjian,
    Shannon McWeeney, Robert Hitzemann. Research Society on Alcoholism
    Annual Meeting. 2009.

11. *A computational framework for the detection of alternative exon
    usage using Affymetrix Exon arrays.* **Ted Laderas**, Michael
    Mooney, Nikki Walter, Kristina Vartanian, Christina Harrington,
    Priscila Darakijan, John Belknap, Robert Hitzemann, and
    Shannon McWeeney. Intelligent Systems in Molecular Biology
    conference, Toronto, July 2008.

12. *Cerebrospinal fluid (CSF) proteomics in children with acute
    lymphoblastic leukemia (ALL).* Robert C Trueworthy, Linda C Stork,
    Yanping Zhong, Sharon Pine, Yousif Matloub, **Ted Laderas**, Shannon
    McWeeney, and Guang Fan. Poster 2006.

13. *A metrics-based low-level preprocessing framework for
    proteomics data.* **Ted Laderas**, Solange Mongoue-Tchokote, Veena
    Rajaraman, Kimberly Sellers, Cory Bystrom and Shannon McWeeney.
    Bioconductor Developer's Meeting, Seattle, Washington, August 2006.

14. *A diagnostic test suite for the RProteomics statistical routines.*
    **Ted Laderas**, Solange Mongoue-Tchokote, Veena Rajaraman, Kimberly
    Sellers and Shannon McWeeney. caBIG annual meeting, Washington DC,
    April 2006.

15. *Consense-Cluster: a framework for comparing and benchmarking
    clustering algorithms.* **Ted Laderas** and Shannon McWeeney.
    Bioconductor Developer's Meeting, Seattle, Washington, August 2005.

**TEACHING STATEMENT**

Effective communication, hands-on learning, and mentoring are core to my
teaching philosophy. I am a strong believer in enabling students in
self-directed learning through hands-on workshops including both
practical skills in bioinformatics (software development, design, and
data analysis and visualization) and social skills essential to
collaboration (interpretation and communication). In terms of practical
skills, I have developed and contributed to a number of courses
(Analytics, RBootcamp, Network Analysis, and BD2K skills courses) and
workshops in a large variety of software development and communication
topics. These topics include software design, interactive visualization,
exploratory data analysis, and presentation of results. In terms of
fostering social and collaborative skills in students, I am a founder
and co-leader of [BioData-Club](https://biodata-club.github.io), a collaborative
forum for students and postdocs to share and
teach each other practical skills for success in a research environment.
Through BioDSP, I have informally mentored a number of students through
feedback and collaboration on workshops and presentations. I believe
that by "training the trainers", we can more effectively educate
students in these skills in Data Science and Analytics. In order to
accomplish this training of trainers, my course materials are openly
available for use and modification by other teachers. Because of my
dedication to teaching and incorporation of student feedback, I have
consistently high ratings from students in terms of teaching both
long-form courses and short-term workshops.

**CURRENT ACADEMIC COURSES**

  + *BMI569/669 Data Analytics*, co-taught with Kaiser Permanente (Brian Sikora and Delilah Moore). Summer Quarter, 2015-Present.
  + *NEU640 Python Bootcamp for Neuroscientists.* Taught with Stephen David and Brad Buran. Winter Quarter, 2018.
  + *BMI551/561 BCBII Statistical Methods in Computational Biology*. Co-instructor. Winter Quarter, 2014-Present.
  + *HSMP/PHE410 Introduction to Health Informatics.* Course Co-Director. Spring Quarter, 2018.

**TEACHING EXPERIENCE**

Lectures and Interactive Workshops are linked where possible.

1. [*Python Bootcamp For Neuroscientists*](https://github.com/dasaderi/python_neurobootcamp). Ted Laderas, Brad Buran, Daniela Sadieri, Charles Heller, Michael Mooney, Lisa Karstens, and Stephen David. 5 day in-person workshop for introductory Python using Neuroscience data.

2. [*Actionable Gene Variants*](https://laderast.github.io/BMI523slides/). Guest Lecture for BMI 523 Clinical Research Informatics. November 2017.

3. [*Exploratory Data Analysis and Statistics*](https://github.com/laderast/DSIExplore). Workshop with Jessica Minnier for OHSU Data Science Institute 2017.

4. [*R-Bootcamp (tidyverse version on DataCamp)*](https://www.datacamp.com/courses/rbootcamp). Active exercises for learning introductory R using the tidyverse. Written with Jessica Minnier and Chester Ismay. 2017. Code and teaching material available at https://github.com/laderast/RBootcamp

5. [*A gRadual intRoduction to the `tidyverse`*](https://github.com/Cascadia-R/gRadual-intRoduction-tidyverse). Workshop given with Chester Ismay for [Cascadia-R 2017](https://cascadiarconf.com) introducing visualization and data cleaning using the `tidyverse`.

6.  [*Assessing Cardiovascular Risk*](https://github.com/laderast/cvdNight1). 2 night Workshop for Portland State University Students teaching [Exploratory Data Analysis](https://github.com/laderast/cvdNight1) and [Machine Learning](https://github.com/laderast/cvdNight2) on a synthetic patient cohort. With David Dorr. May 2017.

7.  [*An Intro to GitHub Pages*](https://github.com/biodata-club/githubPagesTutorial).
    Workshop given with Robin Champieux and Eric Leung on setting up a personal
    GitHub webpage. April 2017.

8.  [*An Intro To Data Carpentry*](https://github.com/erictleung/tutorial-tidyverse).
    Lecture given with Eric Leung about the tidyverse suite of packages for data
    wrangling and visualization. March 2017.

9.  [*The Magic of Markdown*](https://github.com/laderast/magic-of-markdown).
    Updated for 2017, including examples of using Zotero as citation manager. March 2017. [doi:10.5281/zenodo.495614](https://doi.org/10.5281/zenodo.495614)

10.  [*Shiny Tutorial for CSE631*](https://github.com/laderast/CSE631Shiny). Workshop on
    developing interactive visualizations using Shiny for CSE631 Data
    Visualization course. November 2016. [DOI: 10.5281/zenodo.495621](https://doi.org/10.5281/zenodo.495621)

11.  [*Analytics Course*](https://github.com/laderast/AnalyticsCourse).
    New Course material in SQLite, Logistic Regression and Analysis.
    August 2016. [doi:10.5281/zenodo.495623](https://doi.org/10.5281/zenodo.495623)

12.  [*Exploring the DREAM Viral Respiratory Dataset using
    Shiny*](https://github.com/laderast/DreamEDAShiny). Tutorial in
    Exploratory Data Analysis (EDA) using *data.table* and *Shiny*.
    July 2016.

13.  [*Introduction to Exploratory Data Analysis using
    Shiny*](https://github.com/laderast/shinyEDA) - Interactive workshop
    in using a Shiny Dashboard to conduct EDA on a dataset for BD2K
    Advanced Skills Course. May 2016. [doi:10.5281/zenodo.495618](https://doi.org/10.5281/zenodo.495618)

14.  [*Introduction to Machine Learning using
    Markdown*](https://github.com/laderast/MLtutorial) - Interactive
    workshop using Markdown to explore machine learning algorithms for
    BD2K Advanced Skills Course. May 2016.

15.  *Clustering Algorithms*. Lecture given for Statistical
    Methods class. February 2016.

16.  [*iGraph Tutorial*](https://github.com/laderast/igraphTutorial).
    Introductory lecture for the igraph package in R for
    network analysis. November 2015. [doi:10.5281/zenodo.495616](https://doi.org/10.5281/zenodo.495616)

17.  [*The Magic of
    Markdown*](https://github.com/laderast/magic-of-markdown).
    Introduction to Markdown in both R, GitHub pages, and Pandoc. BioDSP
    October 2015. [doi:10.5281/zenodo.495614](https://doi.org/10.5281/zenodo.495614)

18.  *Pharmacogenomics* Lecture for Analytics Course, August 2015.

19.  [*Introduction to
    Clustering*](http://church.ohsu.edu:3838/laderast/clusteringLecture/)*.*
    Interactive slides for understanding clustering. [doi:10.5281/zenodo.495624](https://doi.org/10.5281/zenodo.495624)

20. [*Introduction to
    ggvis*](https://www.dropbox.com/s/n0unpdkd5r2tdu7/ggvis-Tutorial.html?dl=0).
    Lecture/Workshop given for OHSU Bioinformatics Discussion for
    Students and Postdocs, April 2015.

21. [*Introduction to
    ggplot2*](https://www.dropbox.com/s/efbwx3jjx89folh/ggplot2Intro.html?dl=0)*.*
    Lecture/Workshop given for OHSU Bioinformatics Discussion for
    Students and Postdocs, March 2015. [doi:10.5281/zenodo.495622](https://doi.org/10.5281/zenodo.495622)

22. [*Exacloud
    Tutorial*](https://www.dropbox.com/s/chg6ciknxonp5el/exacloud-tutorial.pdf) -
    A DIY tutorial to running jobs on Exacloud, OHSU's cluster
    computing environment. November 2015.

23. [*Shiny
    Tutorial*](https://www.dropbox.com/s/5ceg6wdrustjoae/tutorialApp.zip) -
    A do it yourself tutorial to try out Shiny, ggplot2, and dplyr for
    interactive graphics. September 2015.
    [doi:10.5281/zenodo.495620](https://doi.org/10.5281/zenodo.495620)

24. *Analytics Course*. Instructor. Hybrid Online/On-campus joint course
    with DMICE and Kaiser Permanente Analytics group. August 2015.

25. *R-Bootcamp*. Massively Open Online Course available at:
    <http://dx.doi.org/10.5281/zenodo.13756>. With contributions from
    Eric Leung, Dian Chase, Tracy Edinger, Clint Olson, and
    Gabrielle Chonoo. 2014-Present

26. [*Your In-silico Lab Notebook: Best Practices
    2015*](https://www.dropbox.com/s/ddyq88gqh8priv8/insilicotalk-2015.pptx?dl=0)*.*
    Lecture/Workshop given for OHSU BioDSP group, January 2015.

27. [*Everything you wanted to know about bioinformatics but were afraid
    to
    ask*](https://www.dropbox.com/s/a4qgwap4jehzs8x/everything-you-wanted-to-know.pptx?dl=0)*.*
    Lecture/Workshop given for OHSU PhD/Postdoc Fellows meeting,
    October 2014.

28. [*List Comprehensions in
    Python*](https://www.dropbox.com/s/1my6l8diprnzg4n/lecture_slides_day7_2012-laderas.pptx?dl=0).
    Lecture given for Bioinformatics Programming and Scripting Course,
    Fall 2012.

29. [*Introduction to Unit
    Testing*](https://www.dropbox.com/s/jdgbb07xydmxah5/lecture_slides_day10_1027-laderas.pdf?dl=0).
    Lecture given for Bioinformatics Programming and Scripting Course,
    Fall 2012.

30. *Introduction to Numerical Python (NumPy).* Lecture given for
    Bioinformatics Programming and Scripting Course, Fall 2012.

31. *Introduction to SciPy.* Lectures given for Bioinformatics
    Programming and Scripting Course, Fall 2012.

32. *An Introduction to ODE Models.* Lecture given for Systems Biology
    Class, 2012.

33. [*Integrating Data for Systems
    Biology*](https://www.dropbox.com/s/ksnogpxoh22eit8/IntegratingDataforSystemsBiology.pptx?dl=0).
    Lecture given for Systems Biology Class, 2012.

34. *Work Smarter, Not Harder: Productivity Tools and You.* Lecture
    given for PhD/Postdoctoral meeting, 2011.

35. [*Bayesian
    Networks*](https://www.dropbox.com/s/xab61u1jo9y9io9/bayesnets-new-laderas.pptx?dl=0).
    Lecture given for Statistical Methods in Bioinformatics Class, 2011.

36. [*Gibbs
    Samplers*](https://www.dropbox.com/s/r2j0oi1madn0t1s/GibbsSampler-laderas.pptx?dl=0).
    Lecture given for Statistical Methods in Bioinformatics Class, 2011.
    [doi:10.6084/m9.figshare.4829530](https://doi.org/10.6084/m9.figshare.4829530)

37. *Workshop on Strings and Matrices in R*. Workshop given for
    Statistical Methods in Bioinformatics class. 2011.

38. *Introduction to R Workshop.* Workshop given for Statistical Methods
    in Bioinformatics class. 2011.

39. [*Extended Dependency
    Analysis*](https://www.dropbox.com/s/yt8dybaap4sn8hy/03-eda-extended-dependency-analysis.pptx?dl=0).
    Presentation given for Information Theory Independent Study 2010.

41. [*Conant's Laws of Information that Govern
    Systems*](https://www.dropbox.com/s/kkv4yv2vmwwdqfa/02-conant-laws-of-information.pptx?dl=0).
    Presentation given for Information Theory Independent Study, 2010.

42. [*Ashby's Law of Requisite Variety and Conant's Information Transfer
    in
    Regulation*](https://www.dropbox.com/s/jz0gthjbd9cqbrt/01-ashby-conant-variety.pptx?dl=0).
    Presentation given for Information Theory Independent Study, 2010.

43. *Lab: Using Consense-Cluster to explore the Bittner dataset.*
    Laboratory given as part of Microarray Analysis Course, OHSU, 2006.

**SERVICE STATEMENT**

I am a strong supporter of service at OHSU and beyond. I currently
participate in the DMICE BCB (Bioinformatics and Computational Biology)
Faculty Division meeting, the DMICE Mentoring committee, and have
participated in the BCB Curriculum Retreat in order to plan upcoming
coursework at DMICE. As a Pacific Islander, I have a unique viewpoint
about diversity and I hope to enhance equal opportunity by participating
in OHSU’s Diversity Advisory Council (DAC). Beyond OHSU, I believe that
we need to increase public engagement of science and increase outreach
and mentoring of next-generation science students, especially from
disadvantaged populations will enable these students to succeed in
STEM-based careers. As a former student of Saturday Academy's scientific
mentoring program, I want to contribute back to this community and
engage potential STEM students through student outreach and mentoring. I
am also involved in outreach through the development of course material
for the Biocatalyst training program through Oregon Bioscience
Association, which provides bioscience training for unemployed or
under-employed professionals.

**SERVICE ACCOMPLISHMENTS**

Co-organizer for [Cascadia R Conference](https://cascadiarconf.com).

Contributor to instructor lesson materials to [Software Carpentry](https://software-carpentry.org).

Editor at Large for [Radian Data](http://radiandata.org/).

Participant/Instructor for [PDX Data Rescue/Open Data Day](https://github.com/daniellecrobinson/Data-Rescue-PDX).

Documentation support for [Science Hack Day PDX 2017](http://portland.sciencehackday.org).

Co-Founder of [BioData Club](https://biodata-club.github.io) group for students and postdocs. Managed speakers, developed free workshops for students, and provided feedback for student presentations. BioData Club is currently at 109 members ranging from OHSU and PSU staff, faculty, students and postdocs, providing a unique opportunity to share information and provide peer mentoring to students.

Developed coursework for unemployed or underemployed IT workers in both biosciences and healthcare for Oregon Bioscience Association.

Reviewer for JAMIA, BMC Bioinformatics, and Neurocomputing, and AMIA Symposium.

My mentees include Jason Li and Eisa Mahyari, both PhD candidates in DMICE.

**WORK EXPERIENCE**

*Assistant Professor*, Department of Medical Informatics and Clinical Epidemiolgy, OHSU, 2017-present.
*Postdoctoral Researcher*, OHSU Knight Cancer Institute, 2014-2017.
*Instructor*, Department of Medical Informatics and Clinical Epidemiology. 2015-Present.  
*NLM Postdoctoral Fellow*, Oregon Health & Science University. 2014-2015.  
*Visiting Scientist*, Sage Bionetworks, Seattle Washington. 2014-2015.  
*NLM Predoctoral Fellow*, Medical Informatics and Clinical Epidemiology, Oregon Health & Science University. 2009–2014.  
*Bioinformatics Developer/Project Manager*, OHSU Knight Cancer Institute, Oregon Health & Science University. 2003-2009.  
*Teaching Assistant/Computer Programmer/Server Admin*, Department of Medical Informatics & Clinical Epidemiology, Oregon Health & Science University. 2001-2002.  
*Research Assistant/Computer Programmer*, Department of Molecular Medicine, Oregon Health & Science University. 1999-2001.
*Research Assistant/Teaching Assistant*, Department of Chemistry, Reed College. 1998  

**Ongoing Research Support**

T15LM009461 (supplement)	
Hersh (PI)	
7/01/1992-6/01/2018

*Biomedical Informatics Research Training at Oregon Health & Science University.* Supplement to develop Data Science Materials for T15 training grant. Role: educational developer and instructor.

1U24TR002306-01	
Haendel (PI)	
9/01/2017-9/01/2019

*A National Center for Digital Health Informatics Innovation*. We propose to create a national network for enabling digital health research, innovation, and continuous improvement. The goal is to use information science to impact the way that health care functions and the lives of those it serves. Role: education, software development, and data management advocate.

1U54CA224019-01	
Tyner (PI)	
10/01/2017-10/01/2022

*Tumor intrinsic and microenvironmental mechanisms driving drug combination efficacy and resistance in AML*. Most patients with acute myeloid leukemia (AML) eventually die when their disease becomes resistant to conventional or even newer treatments. Our proposed studies will shed light on the mechanisms of drug resistance, both within the tumor and in the surrounding environment. This knowledge will help identify more effective therapies — involving combinations of two drugs — that will avoid drug resistance and provide better outcomes for patients with AML. Role: Computational Biologist.

No # assigned	
Druker (PI)	
5/1/2013-5/31/2018
The Leukemia & Lymphoma Society
*Beat AML: Precision Medicine for AML Based on Functional Genomics*. The major goals of this project is to transform our approach to AML treatment through a deeper understanding of the diversity of the underlying molecular causes of disease and to bring targeted therapies to AML patients through 1) understanding the spectrum of genetic lesions and molecular drivers, 2) functionally annotating drug sensitivity, and 3) Initiating clinical trials with combinations of drugs in refractory patients. Role: Computational Biologist.

**Completed Research Support**

T15LM009461 	
Hersh (PI)	
7/01/1992-6/30/2017
*Biomedical Informatics Research Training at Oregon Health & Science University*.
Predoctoral and postdoctoral training grant in biomedical informatics.
Role: Predoctoral Fellow (2009-14); Postdoctoral Fellow (2014-2015)

OPP1131709 	
Lewinsohn (PI)	
10/26/2015-10/31/2017
Bill & Melinda Gates Foundation					
*Targeting MAIT cells for TB vaccines*. This proposal is designed to establish whether or not a vaccine targeting Mucosal Associated Invariant (MAIT) can be used to prevent tuberculosis (TB). Role: Computational Biologist 

BD2K Training Grant 
Dorr, D (Co-PI), Haendel, M (Co-PI), McWeeney, S (Co-PI) 
2015-2017 
*Big Data to Knowledge*. The goal of this project is to develop training materials in data science. Role: educational developer and instructor.

**AWARDS**

NLM Predoctoral Fellowship. 2009-2013.  
NLM Postdoctoral Fellowship. 2014-2015.
